cabal-testsuite/src/Test/Cabal/OutputNormalizer.hs

```diff
@@ -54,6 +54,10 @@ normalizeOutput nenv =
         else id)
   -- hackage-security locks occur non-deterministically
   . resub "(Released|Acquired|Waiting) .*hackage-security-lock\n" ""
+  -- Install path frequently has architecture specific elements, so
+  -- nub it out
+  . resub "^Warning: The directory (.+) is not in the system search path."
+          "Warning: The directory <PATH> is not in the system search path."
```

Comment:
I think this hides a bug which you don't address.

Why `extra-packages` is build as local package, and run from somewhere inside `dist-newstyle` and not installed to the store and run from there.

Or so it looked like in previous CI errors.

I'm :-1: on this change. It shouldn't be required.

Reply:
> Why extra-packages is build as local package, and run from somewhere inside dist-newstyle and not installed to the store and run from there.

Hm, that is not what's happening. `extra-packages` are not treated as local. extra-package are installed globally into the store which displays the store-path in above message. Unfortunately store-paths are platform dependent, hence the above change.

Reply:
I think we should rather teach the store-dir location and replace that with `<STOREDIR>` (as we already do for other platform dependent things), rather then hiding big parts of the output

Reply:
So if path is 

```
.../store/ghc-8.6.5/Agda-2.6.2-18fb8b49b1f47de3065c0ae3a191d1060c3959e9293aeb9e2aa88c3705c7a92d/bin/agda
```

we know how to replace `ghc-8.6.5`, matching against `/package-id-hash/` and replacing just `hash` part with `<HASH>` should work. An approximation of package-id is not tricky. See `unqual-name` in https://cabal.readthedocs.io/en/3.4/buildinfo-fields-reference.html#non-terminals

Reply:
Right, for other store path occurrences we are using the same patterns: 

```
    -- Install path frequently has architecture specific elements, so
    -- nub it out
  . resub "Installing (.+) in .+" "Installing \\1 in <PATH>"
```